src/components/Accordion.tsx

```diff
@@ -0,0 +1,77 @@
+import { DummyData, Group } from "@/services/usos/getGroups";
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "@/components/ui/accordion";
+
+type CombinedProps = {
+  index: number;
+  onClick: (index: number, group: Group) => void;
+} & DummyData;
+
+export default function GroupsAccordion({ index, onClick, ...props }: CombinedProps) {
```

Comment:
przenieś typy tutaj i nie twórz osobnego typu

Reply:
wywal domyslny export 
```suggestion
export  const GroupsAccordion = ({ index, onClick, ...props }: CombinedProps) => {
```